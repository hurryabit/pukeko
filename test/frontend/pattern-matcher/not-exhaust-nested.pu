data Pair a b = | Pair a b
data Choice a b =
  | First a
  | Second b
f : Pair (Choice a b) c -> a
f xyz =
  match xyz with
  | Pair (First x) z -> x
