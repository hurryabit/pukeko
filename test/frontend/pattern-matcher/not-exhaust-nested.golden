test/frontend/pattern-matcher/not-exhaust-nested.pu:7:3: unmatched constructor: Second
